components/proxy/src/main/java/com/hotels/styx/infrastructure/FileBackedRegistry.java

```diff
@@ -59,6 +62,7 @@ private HashCode md5(Resource resource) {
     protected void startUp() {
         LOG.info("starting {}", getClass().getSimpleName());
 
+        fileContent = readFileAsString();
```

Comment:
You can make your unit tests much simpler by extracting the file reading into a separate class and injecting it in.